meshroom/nodes/aliceVision/ImageProcessing.py

```diff
@@ -396,6 +398,45 @@ class ImageProcessing(desc.AVCommandLineNode):
             description='If a color profile database is specified but no color profile is found for at least one image, then an error is thrown',
             value=True,
             uid=[0],
+            enabled=lambda node: (node.rawColorInterpretation.value=='DCPLinearProcessing') or (node.rawColorInterpretation.value=='DCPMetadata'),
+        ),
+        
+        desc.BoolParam(
+            name='useDCPColorMatrixOnly',
+            label='Use DCP Color Matrix Only',
+            description='Use only the Color Matrix information from the DCP and ignore the Forward Matrix.',
+            value=True,
+            uid=[0],
+            enabled=lambda node: (node.rawColorInterpretation.value=='DCPLinearProcessing') or (node.rawColorInterpretation.value=='DCPMetadata'),
+        ),
+        
+        desc.BoolParam(
+            name='doWBAfterDemosaicing',
+            label='WB after demosaicing',
+            description='Do White Balance after demosaicing, just before DCP profile application',
+            value=False,
+            uid=[0],
+            enabled=lambda node: (node.rawColorInterpretation.value=='DCPLinearProcessing') or (node.rawColorInterpretation.value=='DCPMetadata'),
+        ),
+        
+        desc.ChoiceParam(
+            name='demosaicingAlgo',
+            label='Demosaicing Algorithm',
+            description='LibRaw Demosaicing Algorithm\n',
+            value='AHD',
+            values=['linear', 'VNG', 'PPG', 'AHD', 'DCB', 'AHD-Mod', 'AFD', 'VCD', 'Mixed', 'LMMSE', 'AMaZE', 'DHT', 'AAHD', 'none'],
+            exclusive=True,
+            uid=[0],
+        ),
+        
+        desc.ChoiceParam(
+            name='highlightMode',
+            label='Highlight mode',
+            description='LibRaw highlight mode\n',
```

Comment:
Add the description for each code (or an enum).
No need for "\n" if there is only one line.